mem/vm/mmu/mmu.go

```diff
@@ -346,6 +365,28 @@ func unique(intSlice []uint64) []uint64 {
 	return list
 }
 
+func (m *middleware) createDefaultPage(pid vm.PID, vAddr uint64, deviceID uint64) vm.Page {
+	// Align virtual address to page boundary
+	alignedVAddr := (vAddr >> m.log2PageSize) << m.log2PageSize
+	pageSize := uint64(1) << m.log2PageSize
+	
+	// For now, we use the virtual address as physical address
+	// In a real system, this would come from a physical memory allocator
+	pAddr := alignedVAddr
```

Comment:
We need to find a page that is not yet used. We want the physical page as small as possible. Do not make it the same as the virtual address.